tests/acceptance/02_classes/02_functions/isnewerthantime.cf

```diff
@@ -0,0 +1,64 @@
+body common control
+{
+  inputs => { "../../default.cf.sub" };
+  bundlesequence => { default("$(this.promise_filename)") };
```

Comment:
Since you are not using anything from _default.cf.sub_ you can consider not including it and instead write your own bundle sequence.

```suggestion
  bundlesequence => { "check" };
```

Reply:
Alternatively, you can omit body common control and just have bundle agent __main__ and then use methods promises to call each of the other bundles you want.

```
bundle agent __main__
{
  methods:
    "init";
    "test";
    "check";
    "cleanup";
```

So, many ways to skin the cat.